bootstrapper/internal/etcdio/etcdio.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+// The etcdio package provides utilities to manage etcd I/O.
+package etcdio
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+	"os"
+	"path"
+	"strconv"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+var (
+	// ErrNoEtcdProcess is returned when no etcd process is found on the node.
+	ErrNoEtcdProcess = errors.New("no etcd process found on node")
+	// ErrMultipleEtcdProcesses is returned when multiple etcd processes are found on the node.
+	ErrMultipleEtcdProcesses = errors.New("multiple etcd processes found on node")
+)
+
+const (
+	// Tells the syscall that a process' priority is going to be set.
+	// See https://elixir.bootlin.com/linux/v6.9.1/source/include/uapi/linux/ioprio.h#L54.
+	ioPrioWhoProcess = 1
+
+	// See https://elixir.bootlin.com/linux/v6.9.1/source/include/uapi/linux/ioprio.h#L11.
+	ioPrioClassShift = 13
+	ioPrioNrClasses  = 8
+	ioPrioClassMask  = ioPrioNrClasses - 1
+	ioPrioPrioMask   = (1 << ioPrioClassShift) - 1
+
+	targetClass = 1 // Realtime IO class for best scheduling prio
+	targetPrio  = 0 // Highest priority within the class
+)
+
+// Client is a client for managing etcd I/O.
+type Client struct {
+	log *slog.Logger
+}
+
+// NewClient creates a new etcd I/O management client.
+func NewClient(log *slog.Logger) *Client {
+	return &Client{log: log}
+}
+
+// PrioritizeIO tries to prioritize the I/O of the etcd process.
+// Since it might be possible that the process just started (if this method is called
+// right after the kubelet started), it retries to do its work each second
+// until it succeeds or the timeout of 10 seconds is reached.
+func (c *Client) PrioritizeIO() error {
```

Comment:
```suggestion
func (c *Client) PrioritizeIO() {
```
Lets just drop any errors in this function.
Setting IO priority is not critical for the functionality of the cluster, and it clears up the caller side, which doesnt return any errors anyway.